src/lib/components/forms/form/form-components/suggestion/suggestion.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { TerraPlacementEnum } from '../../../../../helpers';
+import { TerraSuggestionBoxValueInterface } from '../../../suggestion-box/data/terra-suggestion-box.interface';
+import { merge, noop, Observable, Subject } from 'rxjs';
+import { map, startWith } from 'rxjs/operators';
+import { ControlValueAccessor, FormControl } from '@angular/forms';
+import { SuggestionInterface } from './suggestion.interface';
+import { MatAutocompleteSelectedEvent } from '@angular/material/autocomplete';
+
+/** A type guard to check whether the given value is of type {@link TerraSuggestionBoxValueInterface}. */
+export function isSuggestionValue(value: unknown): value is TerraSuggestionBoxValueInterface {
+    return typeof value === 'object' && 'caption' in value && 'value' in value;
+}
+
+@Component({
+    selector: 'tc-suggestion',
+    templateUrl: './suggestion.component.html'
+})
+export class SuggestionComponent implements ControlValueAccessor, SuggestionInterface, OnInit {
+    /** Set the label. */
+    @Input()
+    public name: string = '';
+
+    /** If true, a * indicates that the value is required. Default false. */
+    @Input()
+    public isRequired: boolean = false;
+
+    /** If true, the autocomplete-input will be disabled. Default false. */
+    @Input()
+    public set isDisabled(disabled: boolean) {
+        disabled ? this._control.disable() : this._control.enable();
+    }
+
+    /** Set the tooltip. */
+    @Input()
+    public tooltipText: string = '';
+
+    /** Set the tooltip placement (bottom, top, left, right). Default top. */
+    @Input()
+    public tooltipPlacement: TerraPlacementEnum = TerraPlacementEnum.TOP;
+
+    /** A list of options that the user can select from the suggestion box. */
+    @Input()
+    public listBoxValues: Array<TerraSuggestionBoxValueInterface> = [];
+
+    /** A formControl for the handling of the value. */
+    public _control: FormControl = new FormControl({ disabled: this.isDisabled });
```

Comment:
```suggestion
    public _control: FormControl = new FormControl();
```
Das keine Wirkung da der Input `isDisabled` zu dem Zeitpunkt eh immer false ist.